policies/roadmap.md

```diff
@@ -0,0 +1,70 @@
+# OpenSSL project roadmap
+
+### Last modified 21 March 2022
+
+This document is the OpenSSL project roadmap. It is a living document
+and is expected to change over time. Objectives and dates should be
+considered aspirational.
+
+## Objectives
+
+Some of these objectives can be achieved more easily and quickly
+than others.
+
+#### Regular releases
+
+The OpenSSL project wants to move to having a regular release interval
+of approximately six months.
+
+#### QUIC
+
+OpenSSL is taking a multi-staged approach to the implementation of the
+[QUIC] transport protocol:
+
+- For OpenSSL 3.1, the focus is on providing a client side single stream
+QUIC implementation.
+
+- OpenSSL 3.2 will follow approximately six months later implementing more
+of the protocol.
+
+- OpenSSL 3.3 aims to complete the implementation a further six moths later.
```

Comment:
typo: six moths!

Reply:
😆 
See it as fluctuating time...  how long time does it take to search and collect six moths? :wink: